semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -53,6 +53,32 @@ def parse(span_kind_value):
         return kind_map.get(span_kind_value)
 
 
+class InstrumentKind(Enum):
+    COUNTER = 1
+    UP_DOWN_COUNTER = 2
+    HISTOGRAM = 3
+    GAUGE = 4
+
+    @staticmethod
+    def parse(instrument_kind_value):
+        return InstrumentKind.kind_map().get(instrument_kind_value)
+
+    @staticmethod
+    def kind_map():
+        return {
+            "Counter": InstrumentKind.COUNTER,
```

Comment:
I think I'd prefer to also use lowercase in the YAML, which would be consistent with everything else. Thoughts?

Reply:
I wanted to recreate what was already in the spec, and it seems pretty consistently using CamelCase. For example, [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#http-server). So, I've stuck with the CamelCase for now if that's ok. 

I think if we wanted to change it to lower case, that would most likely be a discussion in the spec rather than here. Any thoughts?

Reply:
Sorry, I lost track of this PR.
The YAML property names & enums often don't follow the spec names exactly, they are all lowercase (e.g. `type: event` ). Of course, the markdown output should still be UpperCamelCase.

Reply:
Ok sounds good, in my recent changes I've made it so it's lowercase in the yaml but CamelCase in the markdown.